src/ppom-controller.test.ts

```diff
@@ -114,21 +114,21 @@ describe('PPOMController', () => {
         return Promise.resolve();
       });
       jest.runOnlyPendingTimers();
-      expect(spy).toHaveBeenCalledTimes(7);
+      expect(spy).toHaveBeenCalledTimes(6);
```

Comment:
I think I've seen these values change in a couple of PRs. I'll like to understand why the values keep changing and if there's something to do to make sure we don't have new PRs every time they changed

Reply:
I actually did a refactoring in this PR to reduce number of call to version info file when `usePPOM` is called. That resulted in this number of calls getting reduced.